haris2055/SOEN390
# Feature: DMs

This pull request incorporates implementation towards #127 [US-21] and #128 [US-22] (DM Between Connected Users...) of #74 [EPIC-4] Direct Messaging, implemented by contributors to the [DMs_with_UI](https://github.com/haris2055/SOEN390/tree/DMs_with_UI) branch:

To test:
- be signed in to a valid account
- have accepted connections to other users

You should be able to:
- when visiting a connected user's profile:
  - press the Message button
    - you should be redirected to a new conversation with that user
    - from there, you should be able to send messages to the user

- when performing the above actions towards multiple users, the different conversations should populate the sidebar of the messages window

This pull request does not mark the completion of #74 [EPIC-4] Direct Messaging, as user stories #129 [US-23], #131 [US-25], #132 [US-26] are not included.
